internal/authz/authz.go

```diff
@@ -255,7 +255,9 @@ func (a *ClientWrapper) Delete(ctx context.Context, user string, role Role, proj
 			Object:   getProjectForTuple(project),
 		},
 	}).Execute()
-	if err != nil {
+	if err != nil && strings.Contains(err.Error(), "cannot delete a tuple which does not exist") {
```

Comment:
I assume there's no Error we could match using errors.Is ?

Reply:
That's right :cry: